libparsec/crates/client/src/client/mod.rs

```diff
@@ -44,7 +45,12 @@ use crate::{
 };
 use libparsec_client_connection::AuthenticatedCmds;
 use libparsec_platform_async::lock::Mutex as AsyncMutex;
+
 use libparsec_types::prelude::*;
+pub use recovery_device::{
+    create_device_from_recovery, import_recovery_device, ExportRecoveryDeviceError,
```

Comment:
`create_device_from_recovery` is only used in `libparsec/crates/client/src/client/recovery_device.rs` so it should stay private